strategy_pjt1.py

```
@@ -1,313 +1,612 @@
+        except Exception as e:
+            self.logger.error(f"매수 전략 실패: {e}")
+            self.transaction_logger.log_strategy_execution("buy", "error", f"오류: {e}")
+
+    def calculate_profit_rate(self, symbol, current, avg_buy):
+        try:
+            current = float(current)
+            avg_buy = float(avg_buy)
+            if avg_buy == 0:
+                return 'N/A'
+            return (current - avg_buy) / avg_buy
+        except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)